src/OrchardCore.Modules/OrchardCore.Roles/ViewModels/EditRoleViewModel.cs

```diff
@@ -8,6 +8,7 @@ namespace OrchardCore.Roles.ViewModels
     public class EditRoleViewModel
     {
         public string Name { get; set; }
+        public string RoleDescription { get; set; }
```

Comment:
Why it's not `Description`?

Reply:
Because most everywhere else it's `RoleDescription` or `RoleName` - see the `CreateViewModel you adjusted in your PR

Reply:
Ok, LGTM